src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getDurationMetadata.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
+
+import { getDurationMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getDurationMetadata';
+import { statementDefaults } from './fixtures/statements.fixture';
+
+describe('Retrieve duration metadata from statement', () => {
+  it('should return result duration statement', () => {
+      const expectedEmptyMetadata = false;
+
+      // ----------------------------------------------------------------------------------------
```

Comment:
Is there a need for comments like this?

Reply:
These lines I used as test case delimiters. If you want to split tests even into smaller parts then ok